tests/conftest.py

```diff
@@ -11,6 +11,7 @@
 from lakefs_client.client import LakeFSClient
 from lakefs_client.models import BranchCreation, RepositoryCreation
 
+from lakefs_spec.spec import LakeFSFileSystem
```

Comment:
```suggestion
from lakefs_spec import LakeFSFileSystem
```

Since #30 this is now `from lakefs_spec import LakeFSFileSystem`.

Reply:
Addressed in 4e16c07b50d4051f3e9b4563b1a9bc8142beb6c5